kodilla-stream/src/main/java/com/kodilla/stream/world/World.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.stream.world;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class World {
+    private final List<Continent> continentList;
+
+
+    public World(final List<Continent> continentList){
+        this.continentList = continentList;
+    }
+
+    public BigDecimal getPeopleQuantity(){
+        return continentList.stream()
+                .flatMap(continent -> continent.getCountriesList().stream())
+                .distinct()
```

Comment:
czy distinct jest tutaj potrzebne ? Funkcja usuwa potworzenia. To mogloby sie przydac gdybys mial np Turcje w liscie krajow Azji i Europy